### ISSUE TITLE
Use of get_coco_category_maps during training

### ISSUE THREAD

**Issue by p3jawors** [pierluigiferrari/ssd_keras#296]
Hi, I've been trying to go through your ssd300_training script with the MS-COCO dataset. I went through and made all of the noted changes you had in the comments for switching from PASCAL-VOC to MS-COCO. I noticed that in your `eval_utils` there is a `coco_utils.py` script that notes issues with the labelling of the `MS-COCO` dataset due to the 80 categories being spread over 1-90.

I ran through 120 epochs of training with  results
`Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.035`
After further reading I see that the original MS-COCO results for ssd300 seem to have gone through 400k steps instead of 120k, as the default is set to. I also did not change the learning rate from the defaults, so I imagine the combination of these two is the reason for the poor performance. However, before starting another round of training I was wondering if I should be using `coco_utils.get_coco_category_maps()` during training?

**Comment by Shawn0Hsu** [comment]
Hello, I plan to train SSD 300 in COCO data set, but there is something wrong with the train file, could you please send me your train file? Thank you！ @p3jawors
